draft-ietf-rats-msg-wrap.md

```diff
@@ -667,19 +662,15 @@ For example, a `cbor-record` containing an Unprotected CWT Claims Set (UCCS) {{-
 
 {{crypto}} describes a number of methods that can be used to add cryptographic protection to CMW.
 
-## Collection CMWs {#seccons-coll}
+## Using Collection CMWs for Evidence of Composite or Layered Devices {#seccons-coll}
 
-If the Collection CMW is not protected from tampering by external security measures (such as object security primitives) or internal mechanisms (such as intra-item binding), an attacker could easily manipulate the Collection's contents.
-It is the responsibility of the Attester who creates the Collection CMW to ensure that the contents of the Collection are integrity-protected.
-The designer of the attestation technology is typically in charge of ensuring that the security properties are met, not the user of the conceptual message wrapper.
-In particular, when a CMW is used to carry multiple Evidence messages for a composite device or layered attestation, there should be strong binding between the Evidence messages within the Collection.
-This binding is needed to prevent attacks where Evidence from a subverted part of the device is replaced by Evidence from a separate non-subverted device.
-The binding of Evidence messages should be some form of attestation.
+When a Collection CMW is used to encapsulate Evidence for composite or layered attestation of a single device, all Evidence messages within the CMW must be cryptographically bound together to prevent an attacker from replacing Evidence from a compromised device with that from a non-compromised device.
```

Comment:
Doesn't line 670 already say this but in a more succinct way?

Reply:
Yes, L673 concisely reiterates this.